test/integration/graql/query/GraqlDefineIT.java

```diff
@@ -296,109 +121,20 @@ public void whenDefiningARuleUsingCoreAPI_ruleIsPersistedCorrectly(){
         tx.execute(ruleDefinition);
     }
 
-    @Test
-    public void whenDefiningAttributeTypeWithoutValueType_weThrow() {
-        exception.expect(GraqlSemanticException.class);
-        exception.expectMessage(
-                allOf(containsString("my-att"), containsString("value"), containsString("attribute"))
-        );
-        tx.execute(Graql.define(type("my-att").sub(Graql.Token.Type.ATTRIBUTE)));
-    }
-
-    @Test
-    public void whenDefiningRecursiveTypes_weThrow() {
-        exception.expect(GraqlSemanticException.class);
-        exception.expectMessage(allOf(containsString("thingy"), containsString("itself")));
-        tx.execute(Graql.define(type("thingy").sub("thingy")));
-    }
-
     @Test
     public void whenDefiningAnOntologyConceptWithoutALabel_Throw() {
         exception.expect(GraqlSemanticException.class);
         exception.expectMessage(allOf(containsString("entity"), containsString("type")));
         tx.execute(Graql.define(var().sub("entity")));
     }
 
-    @Test
-    public void whenATypeHasNonExistentResource_weThrow() {
-        exception.expect(GraqlSemanticException.class);
-        exception.expectMessage("nothing");
-        tx.execute(Graql.define(type("blah this").sub("entity").has("nothing")));
-    }
-
     @Test
     public void whenDefiningMetaType_Throw() {
```

Comment:
this is migratable - this is trying to define a subtype of "thing" directly, which is not allowed :)

Reply:
Ok, done!

The new test is: `Scenario: define new meta-type ('sub thing') throws`